### PR TITLE
Fixed Dutch Translation

### PR DESCRIPTION
**Pull request by hoptimtom**
# SCP:SL Translation PR Template

## Language Name
Dutch

## State Type of change (New language, changed existing Etc.)
Changed Existing

# Checklist:
- [x ] I have tested the translation changes ingame
- [x ] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x ] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x ] I have made sure that I'm **not** submitting a duplicate translation 
- [x ] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x ] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

**Comment by hoptimtom** [comment]
I guess this is important: I don't know why, but the bodies show up with the english death reason. This might be an issue with the beta/the way I was testing my translation though, as i'm pretty sure i've looked through all the files by now.

**Comment by Dat-Joseph** [comment]
We already know about the Death Cause translation errors. Simply said, the translations happen serverside. I do not know why but they are. Rest assured though that this will not be like that forever.

File: Translations/nl/Abilities.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Ability Niet Gevonden:Ability ID niet gevonden.
+Sprint:Sneller dan lopen, maar maakt meer geluid en heeft stamina nodig.
```
Comment (RinTheWeeb) [inline Translations/nl/Abilities.txt:2]: "gebruikt meer stamina"

File: Translations/nl/Abilities.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Ability Niet Gevonden:Ability ID niet gevonden.
+Sprint:Sneller dan lopen, maar maakt meer geluid en heeft stamina nodig.
+Sluipen:Je bewegingen zijn helemaal stil, maar je beweegt veel slomer
+Inventaris:Open je inventaris en selecteer een voorwerp met de linkermuisknop, of drop het met de rechtermuisknop.
```
Comment (RinTheWeeb) [inline Translations/nl/Abilities.txt:4]: i.p.v drop -> laat het vallen (iets in die richting)

File: Translations/nl/Abilities.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Ability Niet Gevonden:Ability ID niet gevonden.
+Sprint:Sneller dan lopen, maar maakt meer geluid en heeft stamina nodig.
+Sluipen:Je bewegingen zijn helemaal stil, maar je beweegt veel slomer
+Inventaris:Open je inventaris en selecteer een voorwerp met de linkermuisknop, of drop het met de rechtermuisknop.
+Ongewapend:Je hebt niks. Vind iets. Wat dan ook.
+Toegangs Kaart:Je hebt een kaart om deuren mee open te maken. Het is niet de beste, maar het is beter dan niks.
+Standaard Pakket:Je hebt een Geweer, Granaten, een Radio, een Disarmer en een Toegangs Kaart gekregen van de Foundation om je doel te bereiken.
+Bewapend en Gevaarlijk:Je hebt een Lichte Machine Geweer, een Hacking Apparaat (toegangs kaart), en een EHBO doos bemachtigt om je doel te bereiken.
```
Comment (RinTheWeeb) [inline Translations/nl/Abilities.txt:8]: Lichte Machine Geweer -> "licht machinegeweer" + zonder hoofdletters
"hack apparaat" zonder hoofdletters

**Comment by MichalPetryka** [comment]
```
File Badges.txt has 38 lines, expected 44
File Class_Names.txt has 17 lines, expected 18
File Connection_Info.txt has 8 lines, expected 10
```
Please add missing lines

**Pull request merged**